.gitignore

```diff
@@ -1 +1,3 @@
 /node_modules/
+/cypress/screenshots/
+/cypress/videos/
```

Comment:
Ignoring these still allows me to upload them as CI artifacts, which is really what I want, for now. Maybe will start to keep screenshots after integrating Percy or other visual regression testing, but this works for now.